viewing_party/party.py

```diff
@@ -1,23 +1,173 @@
+
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    
+    if title and genre and rating:

```

Comment:
Love the clear code! For further consideration, how could we re-write the if-statement such that we can move the dictionary creation code outside of an if or else block? What's another way we could initialize our movie dictionary?

Reply:
I could move line 7, movie_dict = {}, out of the if statement and put it above on line 5. I think this would help with readability.